artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/BindingsImpl.java

```diff
@@ -53,7 +54,13 @@
 
    private final Map<SimpleString, Integer> routingNamePositions = new ConcurrentHashMap<>();
 
-   private final Map<Long, Binding> bindingsMap = new ConcurrentHashMap<>();
+   private final Map<Long, Binding> bindingsIdMap = new ConcurrentHashMap<>();
```

Comment:
Why not using a primitive version of this map? We have somewhere LongConcurrentHashMap (used into the journal too)

Reply:
It won't use Long instances by just primitive longs, so just footprint: if you know that the map won't be that big i suppose that's fine anyway

Reply:
Got it. FWIW, I just changed the name of the variable here to differentiate it from the new one I added. Given it's been working fine (as far as we know) so far I'd say it should stay the same and we can change it in another PR if necessary.